### ISSUE TITLE
Teleports to house "You're already there!"

### ISSUE THREAD

**Issue by ryant704** [PTYB/SlayerIssueTracker#211]
**Issue:** Game stating you are already at your house when you aren't. This happens at various places around the game at the moment. (Recent game update broke this with the POH update I believe)
**Current behavior:** Gets stuck on the dialogue (see picture)
**Expected behavior:** Teleport to house
**Screenshots/Video:** https://i.postimg.cc/h431LPJD/image.png
**Master:** Nieve
**Task:** Greater Demon (Happens in all random parts of runescape at the moment)
**Logs:** https://pastebin.com/mpPNjPFN
**Additional information:**

**Comment by PTYB** [comment]
Are you banking? Killing? How big is house etc.

**Comment by ryant704** [comment]
It be banking and killing, happens in random spots across the game. 

House: https://i.postimg.cc/j5GvpKYV/image.png

**Comment by PTYB** [comment]
Hmm not a massive house, can  you give me the spots? Any chance theres a house portal there?

**Comment by ryant704** [comment]
Hello, no, this is a game bug. I've just actually googled it because I thought it was listed in the game's bug list but wasn't. Apparently it happens that you have default build mode on.

https://www.reddit.com/r/2007scape/comments/186mr00/why_does_it_keep_saying_youre_already_there_when/

Bottom comment: I've disabled it now. I really should have done that earlier, thinking about it. Cheers for the help, though. I appreciate your time.
